conjure-undertow-lib/src/main/java/com/palantir/conjure/java/undertow/lib/UndertowRuntime.java

```diff
@@ -32,4 +32,8 @@
 
     /** Provides the {@link AuthorizationExtractor} used to read auth tokens from request headers. */
     AuthorizationExtractor auth();
+
+    /** Provides the {@link MarkedParam} used to store safe parameters (header, query, body params) for further
+     * processing. (e.g logging) */
+    MarkedParam markedParam();
```

Comment:
Thoughts on renaming this method to `markers()`?

Reply:
What about other types of elements that can be marked but that aren't concerned by it. Isn't it ambiguous to call this only marker?

Reply:
I think this interface can support all markers, since they exist to provide metadata there's not a large difference between the marker types.

Reply:
Sure but then we should probably either pass the marked element type to the mark method or provide a markParam, markX, etc.

Reply:
Would you be satisfied with MarkedParam -> Markers, mark -> markParam ?

Reply:
That makes sense, though I think it may be cleaner to leave `mark` as is since marking other types of data would use a different method signature. I don't have a strong opinion, `markParam` is also reasonable.